data/core.yaml

```diff
@@ -801,6 +801,11 @@ en:
     field_source: add source
     field_source_label: Source for {field_name}
     field_source_placeholder: URL, newspaper article, book...
+    source:
+      main_input: General source
```

Comment:
From [the forum discussion](https://forum.openhistoricalmap.org/t/updating-the-sources-field-to-edit-source-tags/245/6?u=minh_nguyen), it sounds like we want to steer mappers toward the other fields instead of this one. The placeholder text could suggest that this is unstructured, for example “Description”.